### ISSUE TITLE
[Minor-change] [file="README.md"]  Use of 'sudo' by the root user.

### ISSUE THREAD

**Issue by vharsh** [fedora-cloud/Fedora-Dockerfiles#230]
The __\#__ before the command `sudo dnf install docker fedora-dockerfiles` shows that it was performed by the root user. 
Following the best practices of _not logging in as the root user_, shouldn't the  __\#__  be changed to a **$**  .

**Comment by aditya-konarde** [comment]
+1. It doesn't make sense for the root user to use sudo for dnf. We can either execute the command directly as root or replace # with $ over there. Is anyone working on this?

**Comment by mattdm** [comment]
I agree with @vharsh on keeping with the don't-log-in-as-root practice and using `$ sudo` as the "do with privs" indicator rather than `#`.

**Comment by aditya-konarde** [comment]
I would like to work on this, may I? @mattdm

**Comment by trishnaguha** [comment]
@aditya-konarde Feel free to send the patch in a PR :).

**Comment by aditya-konarde** [comment]
Looks like I sent the PR to my own fork first(new to git). Anyway the pull request is up for review at https://github.com/fedora-cloud/Fedora-Dockerfiles/pull/243. Hopefully it will be my first contribution :D
